test/parallel/test-argv0-property.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+require('../common');
+const assert = require('assert');
+
+assert.ok(process.hasOwnProperty('argv0'), 'argv0 property exists');
+assert.throws(() => {
+  process.argv0 = 'anything';
+}, /read only/);
```

Comment:
Can we have `"TypeError: Cannot assign to read only property 'argv0' of object '#<process>'"` since that way we assert `TypeError` as well (or a truncated anchored RegEx `/^TypeError: `...)